rest/data/subscription.go

```diff
@@ -105,18 +75,20 @@ func SaveSubscriptions(owner string, subscriptions []restModel.APISubscription,
 }
 
 func isEndTrigger(trigger string) bool {
-	return trigger == event.TriggerFailure || trigger == event.TriggerSuccess || trigger == event.TriggerOutcome
+	return trigger == event.TriggerFailure || trigger == event.TriggerSuccess || trigger == event.TriggerOutcome || trigger == event.TriggerFamilyOutcome
 }
 
-func getVersionChildren(versionId string) ([]string, error) {
-	patchDoc, err := patch.FindOne(patch.ByVersion(versionId))
-	if err != nil {
-		return nil, errors.Wrapf(err, "finding patch for version '%s'", versionId)
-	}
-	if patchDoc == nil {
-		return nil, errors.Wrapf(err, "patch for version '%s' not found", versionId)
+func triggerToFamilyTrigger(t string) string {
+	var trigger string
```

Comment:
We can use `default` keyword after all of the cases and have a return statement on each case here instead of declaring a var and returning after the switch is complete

Reply:
done